contracts/test/VTokenHarness.sol

```diff
@@ -12,6 +12,35 @@ contract VTokenHarness is VToken {
 
     mapping(address => bool) public failTransferToAddresses;
 
+    function initializeHarness(
```

Comment:
We removed these items [here](https://github.com/VenusProtocol/isolated-pools/pull/181/files#diff-35683b5d7bfaed24c286787c300ea5bf1baf9fd0fc3c12ad5a019d6a56908d51). Do we need them?

Reply:
I am calling the `initialise` directly from tests without a wrapper. @0xlucian so during rebase you can do the same. Should be straightforward.